src/runtime.js

```diff
@@ -262,8 +262,13 @@ function adoptShared(global, callback) {
   }
 
   // Some deferred polyfills.
-  scheduleInObUpgradeIfNeeded(global);
-  scheduleResObUpgradeIfNeeded(global);
+  const extensionsFor = Services.extensionsFor(global);
```

Comment:
Clever way to avoid Services for now ⭐

Reply:
Yeah! I figure this barely is owned by polyfills themselves, and this is literally the only place those methods were used, so it seemed like an efficient way to untangle some AMP dependencies